tests/e2e/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -6,6 +6,13 @@ import {
 } from '../../utils/helpers';
 import { AdminFlow } from '../../utils/flows';
 
+/**
+ * According to this github issue https://github.com/smooth-code/jest-puppeteer/issues/324, the default timeout isn't
+ * respected and actions timeout after 500ms which cause a lot of test failures. The below lines are a workaround.
+ */
+const setDefaultOptions = require( 'expect-puppeteer' ).setDefaultOptions;
+setDefaultOptions( { timeout: 30000 } );
```

Comment:
Hmm, 30s might be a bit long, but we can try it. Are we seeing timeouts on steps that don't involve installing the plugin? If not, we might want to just consider increasing it there (from previous 5s to maybe 15s).

Reply:
There are two main issues that I observed. The first was the timeout when trying to install the plugin. The second is an issue where it seems that in the plugin install screen, although the plugin option is present, when `toClick` is called the UI behaves like the plugin was not selected. In WCPC we usually call `waitForNavigation` together with a click but there is nothing here to wait for that signifies that the click was successful.

I used 30s because this is the jest `testTimeout` the Woo library uses and it is also the default value of the puppeteer timeout. The 500ms is coming from the `jest-puppeteer` library due to a bug. I would also be against of setting it specifically to actions that we expect to be heavy mainly because the buggy default of 500ms is too small.

In most cases this timeout value will not come into play as once the expectation is completed it returns immediately. This is under the assumption that most cases will be successes though 😄

Reply:
Yeah, I'm just wondering if something fails early on, the dependant tests will fail very slowly. Let's try it though and see.